live-examples/js-examples/string/string-iterator.html

```diff
@@ -0,0 +1,13 @@
+<pre>
+<code id="static-js">const str = "The quick red fox jumped over the lazy dog's back.";
+
+let iterator = str[Symbol.iterator]();
+let theChar = iterator.next();
+
+while(!theChar.done) {
+  // output each character in the string, one per line
+  console.log(theChar);
```

Comment:
This doesn't output the character. To output the character you need `theChar.value`

Reply:
I had been showing the whole value to make it clear that theChar is actually an object with value and done fields, but I've changed this to just show the value.